ansible_collections/arista/avd/roles/eos_designs/python_modules/underlay/router_isis.py

```diff
@@ -72,7 +71,7 @@ def router_isis(self) -> dict | None:
             # TODO - enabling IPv6 only in SR cases as per existing behavior
             # but this could probably be taken out
             if self._underlay_ipv6 is True:
-                router_isis["address_family"].append("ipv6 unicast")
+                router_isis["address_family_ipv6"] = router_isis["address_family_ipv4"]
```

Comment:
I think it would be better here to just rewrite the ipv6 and ipv4 blocks separately in case we want to enable ipv6 only.

Additionally, here your are copying over the dict object and if we want to make them different at some point it will have unexpected effects - notice how when I add "ADD" to `ipv6` it is also added to `ipv4` because this is the same dict (I know for now it does not matter but we should really have two different objects):

```python
>>> d1 = {"ipv4": {"ffr": True, "blah": 42}}
>>> d1["ipv6"] = d1["ipv4"]
>>> id(d1["ipv6"])
4345528640
>>> id(d1["ipv4"])
4345528640
>>> d1
{'ipv4': {'ffr': True, 'blah': 42}, 'ipv6': {'ffr': True, 'blah': 42}}
>>> d1["ipv6"]["ADD"] = 666
>>> d1
{'ipv4': {'ffr': True, 'blah': 42, 'ADD': 666}, 'ipv6': {'ffr': True, 'blah': 42, 'ADD': 666}}
```

Reply:
I updated the module to build the v6 AF independently, thanks!